### PR TITLE
Add game model

### PR DESCRIPTION
**Pull request by ryanveroniwooff**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,11 +11,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```
Comment (jaredAnders) [inline db/schema.rb]: @ryanveroniwooff I believe you're supposed to remove the conflict area markers. [Check out this link on merging conflicts](https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/). Same goes for lines 16, 18, 23, 30, and 49.

File: db/schema.rb [hunk 2]
```diff
@@ -11,11 +11,23 @@
...
+ActiveRecord::Schema.define(version: 20161206021535) do
```
Comment (jaredAnders) [inline db/schema.rb]: @mscoutermarsh what is the best way to merge the schema version? Simply use the new version id and delete the old? Or delete the schema file, pull it from master, run migrations and push that?

Comment (mscoutermarsh) [inline db/schema.rb]: Yes - Go with the latest version when resolving the conflict. 👍

File: db/schema.rb (hunk 1 above)
Comment (mscoutermarsh) [inline db/schema.rb]: 👍 Once you do that, is good to merge to master

Comment (ryanveroniwooff) [inline db/schema.rb]: Thanks guys!

File: db/schema.rb [hunk 3]
```diff
@@ -11,23 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-<<<<<<< HEAD
-ActiveRecord::Schema.define(version: 20161206021535) do
```
Comment (jaredAnders) [inline db/schema.rb]: @ryanveroniwooff I think this one is newer, looks like the date is 1206, the other one is 1205. Let's keep this and delete the older one.

File: db/schema.rb [hunk 4]
```diff
@@ -11,23 +11,13 @@
...
-=======
 ActiveRecord::Schema.define(version: 20161205164755) do
->>>>>>> master
+
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-<<<<<<< HEAD
-  create_table "games", force: true do |t|
```
Comment (ryanveroniwooff) [inline db/schema.rb]: ok, thank you. Maybe someday I just might finish this minor task haha, sorry guys

Comment (jaredAnders) [inline db/schema.rb]: haha you're 99% of the way there already! the rest of your code is really clean, just making sure you didn't delete all of your hard work by accident 😝

**Pull request merged**